src/vtab/array.rs

```diff
@@ -57,11 +57,7 @@ impl ToSql for Array {
 /// `feature = "array"` Register the "rarray" module.
 pub fn load_module(conn: &Connection) -> Result<()> {
     let aux: Option<()> = None;
-    conn.create_module("rarray", &ARRAY_MODULE, aux)
-}
-
-lazy_static::lazy_static! {
-    static ref ARRAY_MODULE: Module<ArrayTab> = eponymous_only_module::<ArrayTab>(1);
+    conn.create_module("rarray", eponymous_only_module::<ArrayTab>(), aux)
```

Comment:
I don't understand why we can't reuse the same Module anymore !

Reply:
It is the same, that function will always return the same pointer. There's no reason to use lazy_static anymore, it would be redundant. I can keep it if you want, though.

You can still do non-static module by Box::leak(Box::new(module)) too.